bedrock/security/templates/security/bug-bounty/faq.html

```diff
@@ -99,9 +99,9 @@ <h2>Eligible software</h2>
       <p>The primary applications we offer bounties for are the most recent
         version of Firefox or Firefox ESR; Firefox for Android, and Firefox
         for iOS. Bounties may be awarded for Firefox Private Network VPN,
-        Lockwise, Firefox Reality, or other non-Beta end-user products offered
-        by Mozilla; however, whether a bounty is awarded and the amount will
-        be subject to the committee.</p>
+        Lockwise, Firefox Reality, or other non-Beta non-End-of-Life end-user
+        products offered by Mozilla; however, whether a bounty is awarded and
+        the amount will be subject to the committee.</p>
```

Comment:
Move the "non-Beta non-End-of-Life" bit before you start naming products to prevent people arguing it doesn't apply to, say, Lockwise or Firefox Reality if we EOL them.

"Bounties may be awarded for other non-Beta, non-End-of-Life, products offered by Mozilla such as Firefox Private Network VPN, Lockwise, and Firefox Reality; however,..."